src/controllers/historycontroller.js

```diff
@@ -0,0 +1,47 @@
+class HistoryController extends Controller
+{
+    static addEvent(event)
```

Comment:
Perhaps turn this into a generic `log(entry)` method. No need for it to be static, and please don't use the word `event` outside the jQuery/event-binding context :).